<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>801</width>
     <height>521</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>전체쇼핑몰</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>맨투맨</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>셔츠</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>청바지</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>슬랙스</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>코트</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>패딩</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_graph1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>550</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>조회하기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_goback">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>550</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>goback</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
